internal/statesync/reactor.go

```diff
@@ -824,3 +840,28 @@ func (r *Reactor) fetchLightBlock(height uint64) (*types.LightBlock, error) {
 	}, nil
 
 }
+
+func (r *Reactor) GetLogger() log.Logger {
+	return r.Logger
+}
+
+func (r *Reactor) InitStateProvider(
+	state sm.State,
+	cfgRPCServers []string,
+	trustOpt light.TrustOptions,
+	logger log.Logger) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
```suggestion
	logger log.Logger,
) error {

		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Reply:
this should probably be a TODO rather than a background.